user_flair/hero_flair.scss

```diff
@@ -626,4 +626,12 @@
 
 .flair-master-probius::before {
   background-position: 40px -2880px
+}
+
+.flair-genji::before {
+  background-position: 0 -2900px
+}
+
+.flair-master-genji::before {
```

Comment:
I don't like adding rules if we won't use them, but I don't mind hugely. Leaving it in is fine by me.